cmd/operator-sdk/new/cmd.go

```diff
@@ -60,6 +60,8 @@ generates a skeletal app-operator application in $GOPATH/src/github.com/example.
 	newCmd.Flags().StringVar(&depManager, "dep-manager", "modules", `Dependency manager the new project will use (choices: "dep", "modules")`)
 	newCmd.Flags().BoolVar(&skipGit, "skip-git-init", false, "Do not init the directory as a git repository")
 	newCmd.Flags().StringVar(&headerFile, "header-file", "", "Path to file containing headers for generated Go files. Copied to hack/boilerplate.go.txt")
+	newCmd.Flags().BoolVar(&makeVendor, "vendor", false, "Use a vendor directory for dependencies. This flag only applies when --dep-manager=modules (the default). If --no-check is set, vendoring is skipped")
+	newCmd.Flags().BoolVar(&noCheck, "no-check", false, "Do not validate the resulting projects' structure and dependencies")
```

Comment:
Nit: Maybe we can call this `skip-validate` in the same way we have `skip-git-init`.